sklearn/ensemble/tests/test_forest.py

```diff
@@ -1470,6 +1471,31 @@ def test_warm_start_oob(name):
     check_warm_start_oob(name)
 
 
+def check_oob_not_computed_twice(name):
```

Comment:
I can see we there is a convention in this file of having a separate function that the test calls, but I think that adds more indirection. Can you place the body of the function directly in the `test_oob_not_computed_twice` test?